src/ekat/util/ekat_catch_main.cpp

```diff
@@ -64,12 +64,15 @@ int main (int argc, char **argv) {
   }
 
   // Initialize test session (initializes kokkos and print config settings).
-  // Ekat provides a defalt impl, but the user can choose
-  // Ekat provides a defalt one, but the user can choose
+  // Ekat provides a default impl, but the user can choose
   // to not use it, and provide one instead.
-  ekat_initialize_test_session(args.size(),args.data());
+  ekat_initialize_test_session(args.size(),args.data(),(comm.rank()==0));
 
+#ifndef NDEBUG
+  MPI_Barrier(comm.mpi_comm());
```

Comment:
What's the rationale for these barriers?

Reply:
Nvm, I think I get it: you don't want the following lines to be interleaved with the config print above or with other stuff from the test below. Makes sense.